content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: section
+title: Administering Jenkins
+---
+
+== `JENKINS_HOME` directory
+Jenkins needs some disk space to perform builds and keep archives.
+You can check this location from the configuration screen of Jenkins.
+By default, this is set to `~/.jenkins`, but you can change this in one of the following ways:
+
+* Set the `JENKINS_HOME` environment variable to the new home directory before launching the servlet container.
+* Set the `JENKINS_HOME` system property in the servlet container.
+* Set the `JNDI` environment entry `JENKINS_HOME` to the new directory.
+See the link:/doc/book/installing/index[Installing Jenkins] for more about how to do this for your container.
+
+You can change this location after you've used Jenkins for a while, too.
+To do this, stop Jenkins completely, move the contents from the old `JENKINS_HOME` to the new home, set the new `JENKINS_HOME`, and restart Jenkins.
+`JENKINS_HOME` has a reasonably obvious directory structure that looks like the following:
+
+[width="100%",cols="100%",]
+|===
+a|
+....
+JENKINS_HOME
+    +- config.xml     (jenkins root configuration)
+    +- *.xml          (other site-wide configuration files)
+    +- userContent    (files in this directory will be served under your http://server/userContent/)
+    +- fingerprints   (stores fingerprint records)
+    +- nodes          (slave configurations)
+    +- plugins        (stores plugins)
+    +- secrets        (secretes needed when migrating credentials to other servers)
+    +- workspace (working directory for the version control system)
+        +- [JOBNAME] (sub directory for each job)
+    +- jobs
+        +- [JOBNAME]      (sub directory for each job)
+            +- config.xml     (job configuration file)
+            +- latest         (symbolic link to the last successful build)
+            +- builds
+                +- [BUILD_ID]     (for each build)
+                    +- build.xml      (build result summary)
+                    +- log            (log file)
+                    +- changelog.xml  (change log)
+....
+|===
+
+=== Backup and restore
+All the settings, build logs, and artifact archives are stored under the `JENKINS_HOME` directory.
+Simply archive this directory to make a backup.
+Similarly, restoring the data is just replacing the contents of the `JENKINS_HOME` directory from a backup.
+Backups can be taken without stopping the server, but when you restore, please stop the server before doing so.
```

Comment:
If we're going to discuss backups here, shouldn't we at least add the caveat that a backup taken without stopping Jenkins means that you might get some "corrupted" files because you copied them while they were being written?